_includes/footer.html

```diff
@@ -6,10 +6,10 @@
           <ul>
             <li><a href="mailto:{{site.email_contacts.suggestions}}">{{ t.menu.contact_us }}</a></li>
             {% if site.twitter %}
-            <li><a class="twitter-follow-button" href="https://twitter.com/{{ site.twitter }}" data-show-count="false">@{{ site.twitter }}</a></li>
+            <li><a class="twitter-follow-button" href="https://twitter.com/{{ site.twitter }}" data-show-count="false">Twitter: @{{ site.twitter }}</a></li>
```

Comment:
Do you think "Twitter" needs to be translated in any languages? If not, this is fine. But I suspect it might need to be translated, even if only in Chinese, Arabic, etc. If we want to allow this to be translated, it would need to be a reference to a "translation key", like maybe `{{ t.general.twitter }}`? (Which of course would need to be added to sdg-translations)

Reply:
@brockfanning I'm checking translations now - if a translation isn't available for a language should I just add it to sdg-translations as English for those languages that aren't available?